Tailscale/package_routines

```diff
@@ -137,6 +137,4 @@ pkg_install(){
 }
 
 pkg_post_install(){
```

Comment:
Now that this is empty, can we remove it? Or does it need to be here to satisfy other plumbing?